README.md

```diff
@@ -1,36 +1,138 @@
-<h1> !!!This repo is under development and NOT stable!!! </h1>
+<h4 align='center'><i>this repo is under development and not stable!</i></h4>
 
-# napari
-[![License](https://img.shields.io/pypi/l/napari.svg)](https://github.com/napari/napari/raw/master/LICENSE)
-[![Build Status](https://api.cirrus-ci.com/github/Napari/napari.svg)](https://cirrus-ci.com/napari/napari)
-[![Python Version](https://img.shields.io/pypi/pyversions/napari.svg)](https://python.org)
-[![PyPI](https://img.shields.io/pypi/v/napari.svg)](https://pypi.org/project/napari)
-[![PyPI - Downloads](https://img.shields.io/pypi/dm/napari.svg)](https://pypistats.org/packages/napari)
-[![Development Status](https://img.shields.io/pypi/status/napari.svg)](https://github.com/napari/napari)
+<h1 align='center'>napari</h1>
 
----
+<h4 align='center'>multi-dimensional image viewer for python</h4>
 
-napari is a fast multi-dimensional image viewer for Python.
+<p align="center">
+  <a href="https://github.com/napari/napari/raw/master/LICENSE"><img src="https://img.shields.io/pypi/l/napari.svg" alt="License"></a>
+  <a href="https://cirrus-ci.com/napari/napari"><img src="https://api.cirrus-ci.com/github/Napari/napari.svg" alt="Build Status"></a>
+  <a href="https://python.org"><img src="https://img.shields.io/pypi/pyversions/napari.svg" alt="Python Version"></a>
+  <a href="https://pypi.org/project/napari"><img src="https://img.shields.io/pypi/v/napari.svg" alt="PyPI"></a>
+  <a href="https://pypistats.org/packages/napari"><img src="https://img.shields.io/pypi/dm/napari.svg" alt="PyPI - Downloads"></a>
+  <a href="https://github.com/napari/napari"><img src="https://img.shields.io/pypi/status/napari.svg" alt="Development Status"></a>
 
-This project is in the **pre-alpha** stage; **expect breaking changes** to be made from patch to patch.
+</p>
+<br>
```

Comment:
What's with all the bare html instead of markdown?

Reply:
it was to center instead of left-align, which is a weird design thing i was trying :)

but i've since changed my mind, so back to markdown!